modules/monitoring-querying-metrics-for-user-defined-projects-with-mon-dashboard.adoc

```diff
@@ -28,29 +31,29 @@ endif::openshift-dedicated,openshift-rosa[]
 
 .Procedure
 
-. From the *Developer* perspective in the {product-title} web console, select *Observe* -> *Metrics*.
+. In the *Developer* perspective of the {product-title} web console, go to *Observe* -> *Metrics*.
 
-. Select the project that you want to view metrics for in the *Project:* list.
+. Select the project that you want to view metrics for from the *Project:* list.
 
-. Select a query from the *Select query* list, or create a custom PromQL query based on the selected query by selecting *Show PromQL*. The metrics from the queries are visualized on the plot.
+. Select a query from the *Select query* list, or create a custom PromQL query based on the selected query by clicking *Show PromQL*. The metrics from the queries are visualized on the plot.
```

Comment:
Hello, thank you for the information! I will have to raise a different PR for this, because the changes in this PR will not go into the 4.17 branch. So I will raise a new Jira for this

Reply:
I was wondering why this went under the radar, but I see it is because it was worked on by a different team, so thank you very much for bringing this to my attention!

Reply:
Jira for tracking is here: https://issues.redhat.com/browse/OBSDOCS-1527

Reply:
back port to 4.17 is fine, the developer UI change is since 4.17